config/app/build.gradle

```
@@ -29,6 +32,22 @@ android {
 
     buildFeatures {
         viewBinding = true
+        compose true
```

nit: Let's use equals here to make it consistent with the line above it.
```suggestion
        compose = true
```

---

Thanks! Totally missed this